tests/qchem/test_observable_hf.py

```diff
@@ -293,26 +303,98 @@ def test_qubit_observable(f_observable, q_observable):
                 [(-0.25 + 0j), (0.25 + 0j), (-0.25 + 0j), (0.25 + 0j)],
                 [qml.Identity(0), qml.PauliZ(1), qml.PauliZ(1) @ qml.PauliZ(3), qml.PauliZ(3)],
             ),
+            "physicist",
         ),
-    ],
-)
-def test_jordan_wigner(f_obs, q_obs):
```

Comment:
The function `test_jordan_wigner_zero_output` is merged in `test_jordan_wigner`.